packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -0,0 +1,178 @@
+
+### Run ngrok
+
+After installing ngrok, ensure that it's running by creating a "tunnel" into a local port (8082 in this example). If you installed directly into your project folder, sample-app, run the following command in your terminal:
+
+```terminal
+sample-app > ./ngrok http 8082
+```
+
+or if you installed in your system path:
+
+```terminal
+sample-app > ngrok http 8082
+```
+
+or if you used your favorite folder:
+
+```terminal
+sample-app > ~/applications/ngrok http 8082
+```
+
+If you see the following content in your terminal, ngrok is running successfully:
+
+![A screen shot of a terminal that displays an ngrok session status, with online in green. The session status contains urls that tunnel into the local port.](/img/ngrok-and-event-hooks-session-status.png)
+
+> **Note:** Copy the forwarding URL that is available from the ngrok terminal session. For example, `https://2d20-142-126-163-77.ngrok.io`. You'll use this URL when setting up your Okta Event Hook.
+
+See [ngrok](https://ngrok.com) or their [documentation](https://ngrok.com/docs) for further information.
+
+### Review ngrok inspection interface
+
+The ngrok inspection interface provides an opportunity to review all calls to your local application. See [ngrok documentation](https://ngrok.com/docs#inspect-requests) for details on using this interface. With the ngrok utility running, open the following URL in a browser: `http://localhost:4040`.
+
+Each call to your local application appears in the interface and includes the response body, header, and other details:
+
+![A screen shot of the ngrok web interface that includes the response body, header, and other details.](/img/ngrok-and-event-hooks-web-interface.png)
+
+## Create an Okta Event Hook
+
+Create the Okta Event Hook to work with your local application, which can now be exposed externally. The Event Hook must be set up and verified within your Okta Admin Console.
+
+### Set up the Event Hook
+
+1. Sign in to your [Okta org](https://login.okta.com/).
+
+2. From the Admin Console, go to **Workflow** > **Event Hooks**.
+
+3. Click **Create Event Hook**. The **Add Event Hook Endpoint** dialog box opens.
+
+4. In the **Name** field, add a unique name for the Hook (in this example, "New User Event Hook").
+
+5. In the **URL** field, add your external service URL, including endpoint. For this example, use the code endpoint, `/userCreated` from `server.js` with the `https://` URL from the [ngrok session](#run-ngrok). For example, your URL should appear similar to: `https://2d20-142-126-163-77.ngrok.io/userCreated`.
+
+6. Include authentication field and secret. In this example, our `server.js` code uses Basic Authentication:
```

Comment:
Because you refer explicitly to the UI components, should you capitalize and bold?

Reply:
Will do.